_posts/2024-02-21-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Enhancing Security in OpenSearch 2.12.0: The End of Default Admin Password"
+authors:
+ - dchanp
+date: 2024-02-21
```

Comment:
This blog post is intended to be published post 2.12. Current [release window](https://opensearch.org/releases.html) for 2.12 ends on 02/20/24 and hence I've kept this date at 21st. Please help me correct this if anything should be changed.